.circleci/bazel.rc

```diff
@@ -0,0 +1,28 @@
+# These options are enabled when running on CI
```

Comment:
can you also remove the `--config=ci` setup? Both anywhere that is passed and also any config options in bazel.rc?
This setup should be mutually exclusive with using `--config`

Reply:
You mean the `build --local_resources=14336,8.0,1.0`?

Reply:
yeah all of them, we should remove the `build:ci` lines from `tools/bazel.rc` since we now have a separate file for these

Reply:
Done 👍